### DISCUSSION TITLE
Mockups

### DISCUSSION THREAD

**Discussion by ivavay** [mdwiltfong/secret-santa#12] [Ideas]
Hi! I'm coming from a design background so was wondering if it would be helpful to mock up a quick wireframe of the different pages / components we need for how this Secret Santa would work. To get started, I would need to know the decided functionalities that we want in our app.

**Comment by mdwiltfong** [comment]
Hey @ivavay ! 

I've created an issue for this [here](https://github.com/mdwiltfong/secret-santa/issues/13)

**Comment by mdwiltfong** [comment]
I've also created a [wiki](https://github.com/mdwiltfong/secret-santa/wiki/Roadmap) that helps outline aspects of the app. If you have questions please ask them here!

**Reply by ivavay** [reply]
I saw the files under src are in typescript (tsx)? What should I do if I'm not familiar with TS and only with react? I'm gonna start seeing how Reactstrap works and work on the frontend.

**Reply by mdwiltfong** [reply]
Good question! If you look at some of the .tsx files, you will actually see a lot of the content looks very similiar to plain old React. So you don't have to do much. Although we should try to leverage TS as much as we can. 

How about you make as much progress as you can with writting React and this wednesday we can tackle the TS part together. The React Docs actually have a section dedicated to using TypeScript with React. It's a short read! 

https://react.dev/learn/typescript

**Reply by ivavay** [reply]
Okay thanks sure!
